### DISCUSSION TITLE
[Projects] Specify status/labels/assignees when quickly adding issues via Ctrl Space

### DISCUSSION THREAD

**Discussion by jschuur** [community/community#14205] [Projects and Issues] [answered]
Currently, when I’m in a view, I can add a new issue by hitting Ctrl Space and then typing in a title. However, if I’m in a filtered view by any of the search fields, this new entry will disappear, since it doesn’t have the proper attributes for that view and you have to switch views to edit it.

Concrete example: I add something to a ‘Sprint’ view that is filtered by certain statuses, but it starts out with a blank status, so now I have to go dig it out from the unfiltered view.

Ideally, other fields could be provided all in the same line with the title when you hit Ctrl Space. Tricky, since this might require some delimiter being introduced that knows when the title ends and the other fields end. So maybe I type in '“Add cool new feature” Status:Sprint assignee:@me label:feature’, with the title being provided in double quotes.

**Comment by bwells78** [comment]
YES!  This is incredibly frustrating, rendering the + Add Item button more frustrating than helpful.  But it seems like GH can make some accurate assumptions here based on my filter and the column I'm adding to, saving me a lot of time and clicks. 

Even if we had the option to immediately add it a repo rather than having to unfilter-find-open-convert-blah-blah-blah, it'd be great.

**Comment by jschuur** [comment, marked as answer]
It looks like when you're in a status filtered view now and quick add a new issue, it actually defaults it to the status of that view, which solves the original problem nicelty.

More specifically, if you view is filtered by multiple statuses, it sets it to the first status value from your filter (so afilter of 'status:"Sprint","In Progress"' added a status of 'Sprint' for me just now).
